[All Hosts] (troubleshooting) adding cache handling to troubleshootin… (#1421)
OfficeDev/office-js-docs-pr: docs/testing/testing-and-troubleshooting.md

## Changes to static files, such as JavaScript, HTML, and CSS do not take effect

The browser may be caching these files. To prevent this, turn off client-side caching when developing. The details will depend on what kind of server you are using. In most cases, it involves adding certain headers to the HTTP Responses. We suggest the following set:

- Cache-Control: "private, no-cache, no-store"
- Pragma: "no-cache"
- Expires: "-1"

For an example of doing this in an Node.JS Express server, see [this app.js file](https://github.com/OfficeDev/Office-Add-in-NodeJS-SSO/blob/master/Complete/app.js). For an example in an ASP.NET project, see [this cshtml file](https://github.com/OfficeDev/Office-Add-in-ASPNET-SSO/blob/master/src/Office-Add-in-ASPNET-SSO-WebAPI/Views/Shared/_Layout.cshtml).

If your add-in is hosted in Internet Information Server (IIS), you could also add the following to the web.config.

```xml
<system.webServer>
  <staticContent>
    <clientCache cacheControlMode="UseMaxAge" cacheControlMaxAge="0.00:00:00" cacheControlCustom="must-revalidate" />
  </staticContent>
```

> If these steps don't seem to work at first, you may need to clear the browser's cache. Do this through the UI of the browser. Sometimes the Edge cache is not successfully cleared when you try to clear it in the Edge UI. If that happens, run the following command in a Windows Command Prompt.

Reviewer: ```suggestion
If these steps don't seem to work at first, you may need to clear the browser's cache. Do this through the UI of the browser. Sometimes the Edge cache isn't successfully cleared when you try to clear it in the Edge UI. If that happens, run the following command in a Windows Command Prompt.
```

> ```bash

Reviewer: I think instead of bash we are trying to use command line - also do we need to specify "Windows Command Prompt" or can we just say command prompt above?

Author: @mscharlock Do you mean put "commandline" (one word) after the ```?

Reviewer: Yep! I think we agreed to do ```command&nbsp;line```

Author: That wasn't listed in the OPS help as a supported term, so I just went ahead and merged the PR with the "bash" still there.
